src/harness/harness.ts

```diff
@@ -2003,6 +2082,57 @@ namespace Harness {
             const comparison = compareToBaseline(actual, relativeFileName, opts);
             writeComparison(comparison.expected, comparison.actual, relativeFileName, actualFileName);
         }
+
+        export function runMultifileBaseline(relativeFileBase: string, extension: string, generateContent: () => IterableIterator<[string, string, number]> | IterableIterator<[string, string]>, opts?: BaselineOptions, referencedExtensions?: string[]): void {

```

Comment:
looks like return type of `generateContent` would be better as `IterableIterator<[string, string, number | undefined]>`.

Reply:
An `IterableIterator<[string, string]>` is not assignable to an `IterableIterator<[string, string, number | undefined]>`, unfortunately. Tuple optionality would be nice.